Mathlib/Topology/Basic.lean

```diff
@@ -862,14 +862,8 @@ theorem eventually_nhds_iff {p : X → Prop} :
 #align eventually_nhds_iff eventually_nhds_iff
 
 theorem frequently_nhds_iff {p : X → Prop} :
-    (∃ᶠ x in 𝓝 x, p x) ↔ ∀ U : Set X, IsOpen U → x ∈ U → ∃ x ∈ U, p x := by
-  simp only [frequently_iff, mem_nhds_iff, forall_exists_index, and_imp]
-  constructor
-  · intro a b c d
-    exact a b subset_rfl c d
-  · intro a _ c d e f
-    obtain ⟨v, hv, hv2⟩ := a c e f
-    exact ⟨v, d hv, hv2⟩
+    (∃ᶠ y in 𝓝 x, p y) ↔ ∀ U : Set X, IsOpen U → x ∈ U → ∃ y ∈ U, p y :=
```

Comment:
to match the way things unfold better
```suggestion
    (∃ᶠ y in 𝓝 x, p y) ↔ ∀ U : Set X, x ∈ U → IsOpen U → ∃ y ∈ U, p y :=
```

Also, while you're here, can you please fix the statement of `eventually_nhds_iff` with the bounded / free variables the same way I had you do with this one? It's nearly impossible to parse.